docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-kestra.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Deploy with Kestra
+description: How to deploy a pipeline with Kestra
+keywords: [how to, deploy a pipeline, Kestra]
+---
+
+# Deploy with Kestra
+
+## Introduction to Kestra
+
+[Kestra](https://kestra.io/docs) is an open-source, scalable orchestration platform that enables
+all engineers to manage business-critical workflows declaratively in code. By applying 
+infrastructure as code best practices to data, process, and microservice orchestration, you
+can build reliable workflows and manage them.
+
+Kestra facilitates reliable workflow management, offering advanced settings for resiliency,
+triggers, real-time monitoring, and integration capabilities, making it a valuable tool for data
+engineers and developers.
+
+### Kestra features
+
+Kestra provides a robust orchestration engine with features including:
+
+- Workflows are accessible through a user interface, event-driven
+  automation, and an embedded visual studio code editor.
+- It also offers embedded documentation, a live-updating topology view, and access to over 400
+  plugins, enhancing its versatility.
+- Kestra supports Git & CI/CD integrations, basic authentication, and benefits from community
+  support.
+
+To know more, please refer to [Kestra's documentation.](https://kestra.io/docs)
+
+## Building Data Pipelines with `dlt`
+
+**`dlt`** is an open-source Python library that allows you to declaratively load data sources
+into well-structured tables or datasets through automatic schema inference and evolution. It
+simplifies building data pipelines by providing functionality to support the entire extract and load
+process.
+
+### How does `dlt` integrate with Kestra for pipeline orchestration?
+
+To illustrate setting up a pipeline in Kestra, we’ll be using the following example: 
+[From Inbox to Insights AI-Enhanced Email Analysis with dlt and Kestra.](https://kestra.io/blogs/2023-12-04-dlt-kestra-usage)
+
+It demonstrates automating a workflow to load data from Gmail to BigQuery using the `dlt`,
```

Comment:
**The example** demonstrates automating a workflow . . .

Reply:
done